README.md

```diff
@@ -108,17 +108,17 @@ locally in standalone mode (single process).
      2. Follow the instructions in the [Kafka quickstart](https://kafka.apache.org/quickstart)
      to read the message from your Kafka topic.
 
-   - `PubSubLiteSinkConnector`
-     1. Follow the instructions in the [Kafka quickstart](https://kafka.apache.org/quickstart)
-     to publish a message to the Kafka topic.
-     2. [Pull](https://cloud.google.com/pubsub/docs/publish-receive-messages-console#pull_the_message_from_the_subscription)
-     the message from your Pub/Sub Lite subscription.
-
    - `PubSubLiteSinkConnector`
```

Comment:
I believe this needs to be the `PubSubLiteSourceConnector`, not the other way around.